css/logo-nav.css

```diff
@@ -664,7 +829,7 @@ a {
     margin-left: 20px;
     margin-bottom: 10px;
     cursor: default;
-} 
+}
```

Comment:
Can you get rid of the 2 transitions around lines 860 at the same time, they are the only 2 instances 
```
@media (max-width: 395px) {
@media (max-width: 365px) {
```

They seem to be for the "About" page and are broken anyway:

![screen shot 2016-06-22 at 14 22 48](https://cloud.githubusercontent.com/assets/335567/16284187/d9f02156-3884-11e6-9fbb-138c9295ba98.png)